geniza/corpus/templates/corpus/document_detail.html

```diff
@@ -48,6 +48,10 @@ <h3 class="sr-only">
                         {# link ? #}
                     {% endfor %}
                 {% endif %}
+                {% if document.fragments.count > 1 %}
+                    <dt class="secondary">{% translate 'Shelfmark' %}</dt>
+                    <dd class="secondary">{{ document.shelfmark }}</dd>
+                {% endif %}
```

Comment:
per the linked designs shelfmark should be displayed before editor and should not be in the lighter gray color

Reply:
1a3f626 ✅